test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -907,7 +911,9 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 			"spec.unschedulable": "false",
 		}.AsSelector().String()})
 		framework.ExpectNoError(err)
-		framework.ExpectEqual(nodesToBreakCount <= len(nodes.Items), true)
+		if nodesToBreakCount > len(nodes.Items) {
+			framework.Failf("nodesToBreakCount :%v", nodesToBreakCount, "must be less than or equal to len(nodes.Items) :%v", len(nodes.Items))
```

Comment:
You can't have more than one format string.

```suggestion
			framework.Failf("Expected at most %d nodes to break, got %d", len(nodes.Items), nodesToBreakCount)
```